src/gigs-board/components/layout/Page.jsx

```diff
@@ -53,9 +53,9 @@ function href(widgetName, linkProps) {
 /* END_INCLUDE: "common.jsx" */
 
 return (
-  <>
+  <div className="h-100" style={{ minHeight: "86vh" }}>
```

Comment:
Where is 86vh coming from? I see it partially helping, but it seems we need to have a better solution in the future.

Reply:
It's the minimum value that works